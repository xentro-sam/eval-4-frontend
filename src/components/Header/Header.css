#header-title {
    background-color: white;
    font-size: 2vw;
    color: rgb(60, 61, 60);
    font-weight: 600;
    padding: 1vw 3vw;
}