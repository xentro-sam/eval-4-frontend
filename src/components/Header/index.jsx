import * as React from 'react';
import './Header.css';

export default function Header() {
  return (
    <div id="header-title">
        Content Types
    </div>
  );
}
