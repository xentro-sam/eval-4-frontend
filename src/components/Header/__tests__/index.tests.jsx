import React from 'react';
import {render, screen} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from '../index';
import {waitFor} from '@testing-library/dom';
import {act} from 'react-dom/test-utils';

const mockData = {
  title: 'Test Title',
};

describe('ContentType', () => {
  it('should render correctly', async () => {
    const {asFragment} = await act( async () => render(<Header {...mockData} />));
    await waitFor(() => {
      expect(screen.getByText('Test Title')).toBeTruthy();
    });
    expect(asFragment()).toMatchSnapshot();
  });
});
