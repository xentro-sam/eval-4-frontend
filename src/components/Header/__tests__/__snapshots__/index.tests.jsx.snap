// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentType should render correctly 1`] = `
<DocumentFragment>
  <div
    id="header-title"
  >
    Test Title
  </div>
</DocumentFragment>
`;
